org.moreunit.swtbot.test/src/org/moreunit/refactoring/RenameMethodTest.java

```diff
@@ -4,6 +4,9 @@
 
 import org.eclipse.jdt.core.IMethod;
 import org.eclipse.jdt.core.JavaModelException;
+import org.eclipse.jdt.internal.ui.refactoring.RefactoringMessages;
+import org.eclipse.jface.dialogs.IDialogConstants;
+import org.eclipse.ltk.internal.ui.refactoring.RefactoringUIMessages;
```

Comment:
it is internal of Messages, I prefer using internals instead of having silent dependencies on the text. it has also the advantages that it is not language dependent and if the text is modified, the test is still working.